hardhat.config.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
       accounts: { mnemonic: MNEMONIC },
       chainId: ENV_CHAIN_IDS[NETWORK],
       gas: 7700000,
-      gasPrice: 1 * GWEI,
+      gasPrice: 10 * GWEI,
```

Comment:
This change should be on a different PR since it is doing something other than the PR title.

Reply:
Yeah, this isn't even needed, `1 GWEI` is fine on testnet, even if you want to experiment with high gas prices, you can just modify it. Please revert it. @adi44

Reply:
https://github.com/razor-network/contracts/pull/96 @dev1644 it is the latest PR